static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -0,0 +1,171 @@
+@import 'svg';
+
+
+$contribute-yellow: #ffbb00;
+$gu-blue: #005689;
```

Comment:
Shouldn't have to create variables for this, there are already variables for both - http://guardian.github.io/pasteup-palette/demo/